commands/operator-sdk/cmd/up/local.go

```diff
@@ -112,3 +128,37 @@ func upLocal(projectName string) {
 		cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to run operator locally: %v", err))
 	}
 }
+
+func upLocalAnsible() {
+	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
```

Comment:
@shawn-hurley If you don't namespace the manager(or the underlying cache really) then you're running a cluster-scoped operator.

So correct me if I'm wrong but I just noticed that the incoming events will be from all namespaces since the Source is Listing without the namespace option. 
https://github.com/operator-framework/operator-sdk/blob/master/pkg/ansible/controller/source.go#L61

So this isn't really true:
https://github.com/operator-framework/operator-sdk/blob/master/pkg/ansible/controller/controller.go#L50

The two ways you can namespace your watched events are either namespacing the entire manager in which case the cache will only hold objects from your desired namespace. That way you don't need cluster-scoped permissions as well.

Or restrict the namespace with the [ListOptions](https://github.com/kubernetes-sigs/controller-runtime/blob/79f06014d49b565e2d980fcd9519b33874ddb8d6/pkg/client/interfaces.go#L203) so your source only gets events for your namespace. You'll still need cluster-scoped permissions in this case.

Reply:
@hasbro17 Your correct in that the AO is watching all namespaces as of now. This was because your PR had not been merged yet to allow for the restriction to a namespace, and we have not gone back and updated it yet. 

The log statement was not updated, your correct that it is not 100% correct at the moment. 

What we want to make sure is that we do this the correct way. As we are targeting OLM, we want to make sure we understand how they will pass us the namespace(s) that we will need to watch. I have not seen anything that tells me how OLM will do this for sure, and I was waiting on that before implementing.

@dymurray For this PR, I think we may want to tell the manager to restrict to a namespace. This is done by setting the [namespace](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/manager/manager.go#L113) on the options. I assume that we can re-use the env var approach that is used already.

Reply:
Thanks guys, I have updated so that we are setting ListOptions `namespace` field.